documentation/hub/security.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: documentation
+menu: hub_security
+subtitle: Security information for Flyway Hub
+redirect_from: /documentation/hub-security
+---
+
+# How we protect your data
+
+Flyway Hub is a hosted service currently running entirely within AWS infrastructure. We operate Flyway Hub from the London data center. Please <a href="mailto:flywayhub@red-gate.com">contact us via email</a> if you'd like us to operate Flyway Hub in multiple regions.
+
+## Security Measures
+
+All Flyway Hub resources are stored in the Flyway Hub cloud which is a multitenant service. Here’s what we do to protect your data:
+
+* All user data and running instances are isolated from each other through industry-standard approaches:
+  * Process and filesystem containerisation of running workloads
+  * Network isolation between running workloads
+* Connections to the Flyway Hub API use OpenID Connect authentication and JSON Web Tokens to ensure users only see their own projects
+  * Therefore, users may only see and run their own projects
```

Comment:
Do we need to mention these two at all? I'd say this is just a given of using any SaaS product. The ability to only interact with your own stuff isn't really an extra security feature.

Reply:
Think it's fine for now